components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/index.jsp

```diff
@@ -106,8 +106,8 @@
 
     function deleteUserStores() {
         var selected = false;
-        if (Array.isArray(document.userStoreForm.userStores)
-            &&  document.userStoreForm.userStore.length) { // there is more than 1 user store
+        if (document.userStoreForm.userStores != null && document.userStoreForm.userStores.length > 1) {
```

Comment:
```suggestion
        if (document?.userStoreForm?.userStores?.length > 1) {
```
We can use ?. operator as a short circuit where "if the expression up until that point is null or undefined, then the whole expression evaluates to undefined.